src/inputs/input.jsx

```diff
@@ -10,16 +17,23 @@ const propTypes = {
     autoComplete: PropTypes.oneOf(['off', 'on']),
     autoFocus: PropTypes.bool,
     className: PropTypes.string,
+    /**
+     * A DatePickerInput can be disabled.
```

Comment:
Oops ... copy-and-paste error.  Should be "An Input can be disabled."

Reply:
Whoops, I tried to change all those... I guess Eslint fixes finally took over my process. Fixing...

Reply:
Commit 7c8b691 fixes this.

Reply:
Confirmed component name was updated.  Resolving.